workers/unity/Packages/com.improbable.gdk.core/package.json

```diff
@@ -0,0 +1,15 @@
+{
+    "name": "com.improbable.gdk.core",
+    "displayName": "SpatialOS GDK Core",
+    "version": "0.0.1",
+    "unity": "2018.2",
+    "description": "SpatialOS GDK Core Module.",
+    "dependencies": {
+        "com.improbable.gdk.legacy": "0.0.1"
+    },
+    "keywords": [
```

Comment:
What are these keywords used for? I guess for indexing in a package searching function?

Reply:
Not clear what they're used for right now, have removed them.